cmake/LLVMIRUtil.cmake

```diff
@@ -175,16 +175,17 @@ function(llvmir_attach_bc_target)
       continue()
     endif()
 
+    get_target_property(SOURCE_DIR ${DEPENDS_TRGT} SOURCE_DIR)
+    get_filename_component(INFILE ${IN_FILE} ABSOLUTE BASE_DIR ${SOURCE_DIR})
     get_filename_component(OUTFILE ${IN_FILE} NAME)
```

Comment:
What would happen if you used the `INFILE`  here? Would we get the same thing?

Reply:
Yes, I think so. According to [CMake's document](https://cmake.org/cmake/help/latest/command/get_filename_component.html), `get_filename_component(<var> <FileName> NAME)` returns "File name without directory". Therefore, we should get the same result either passing a relative path or a absolute path.

Reply:
Maybe try it to make sure and then you can assign it to the variable with a name that indicates an absolute file path as we discussed below.

Reply:
I just tried, and the results are the same when using `IN_FILE` or the absolute version `ABS_IN_FILE`